translations/messages.fr.xlf

```diff
@@ -0,0 +1,3070 @@
+        <source>Learn</source>
+        <target>Apprendre</target>
+      </trans-unit>
+      <trans-unit id="3318133641595899163" datatype="html">
+        <source>AwesomBoard</source>
+        <target>AwesomBoard</target>
+      </trans-unit>
+      <trans-unit id="3620319853901130962" datatype="html">
+        <source>AwesomBoard is a website to play and learn the rules of various abstract strategy games with perfect information.<x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br/&gt;"/> Such games are games that do not include any randomness, agility, or hidden information. These games are played by two players, turn by turn. </source>
+        <target>AwesomBoard est un site qui permet de jouer et d&apos;apprendre les règles de nombreux jeux de stratégie combinatoire à information parfaite.<x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br/&gt;"/> On comprends donc là dedans les jeux ne faisant intervenir ni hasard, ni agilité, ni informations cachées, et uniquement des jeux deux joueurs et tours par tours! </target>
+      </trans-unit>
+      <trans-unit id="4005579048990110948" datatype="html">
+        <source>It is not your turn.</source>
+        <target>Ce n&apos;est pas votre tour !</target>
+      </trans-unit>
+      <trans-unit id="4691729121764741641" datatype="html">
+        <source>You cannot clone a game. This feature might be implemented later.</source>
+        <target>Clôner une partie n&apos;est pas encore possible. Cette fonctionnalité pour être implémentée dans un futur incertain.</target>
+      </trans-unit>
+      <trans-unit id="3568920234618711065" datatype="html">
+        <source>The game has ended.</source>
+        <target>La partie est terminée.</target>
+      </trans-unit>
+      <trans-unit id="7800061171704298797" datatype="html">
+        <source>Human</source>
+        <target>Humain</target>
+      </trans-unit>
+      <trans-unit id="6063984594211340121" datatype="html">
+        <source>Pick the level</source>
+        <target>Choisissez le niveau</target>
+      </trans-unit>
+      <trans-unit id="8800476882871783599" datatype="html">
+        <source>Level <x id="INTERPOLATION" equiv-text="{{ i }}"/></source>
+        <target>Niveau <x id="INTERPOLATION" equiv-text="{{ i }}"/></target>
+      </trans-unit>
+      <trans-unit id="3272612818120648715" datatype="html">
+        <source><x id="INTERPOLATION" equiv-text="{{ compo.scores[0] }}"/> points</source>
+        <target><x id="INTERPOLATION" equiv-text="{{ compo.scores[0] }}"/> points</target>
+      </trans-unit>
+      <trans-unit id="8739046962840362623" datatype="html">
+        <source><x id="INTERPOLATION" equiv-text="{{ winner }}"/> won</source>
+        <target><x id="INTERPOLATION" equiv-text="{{ winner }}"/> a gagné</target>
+      </trans-unit>
+      <trans-unit id="8647687729200262691" datatype="html">
+        <source>Draw</source>
+        <target>Match nul</target>
+      </trans-unit>
```

Comment:
On a une distinction entre draw quand c'est match-nul d'un commun accord et tie quand la partie termine sans vainqueur ? Si c'est bien ça ?

Reply:
de ce que je trouve, tie = US, draw = reste du monde. Et ils sont synonymes parfaits, SAUF au cricket :D (où draw indique que la limite de temps de partie a été dépassée). Du coup, que faisions nous?

Reply:
J'I truste you, then !
(resolve please)